lib/specs.js

```diff
@@ -193,7 +196,7 @@ function assembleProtocol(fpath, opts, cb) {
  *
  * The parsing logic is as follows:
  *
- * + If `str` contains `path.sep` (on windows `\`, otherwise `/`) and is a path
```

Comment:
Can you elaborate on this change?

Reply:
> I could try to move this function into some Node-specific module or exclude it from the browser build, so that path.sep can be used--does that sound like a better route?

That sounds great. Could you do it in `lib/files.js`?